hashing/src/chunk.rs

```diff
@@ -85,4 +133,44 @@ mod test {
         assert_eq!(proof_0.root_hash(), expected_root);
         assert_eq!(proof_1.root_hash(), expected_root);
     }
+
+    #[test]
+    fn validates_chunk_with_proofs() {
+        let large_data_size = (CHUNK_SIZE * 2) as usize;
```

Comment:
If `CHUNK_SIZE = 1`, is this really large?

Reply:
Meaning: "large enough to not fit into a single chunk".
Renamed to something more meaningful, though.